src/qt/bitcoinunits.cpp

```diff
@@ -40,9 +40,9 @@ QString BitcoinUnits::name(int unit)
 {
     switch(unit)
     {
-    case BTC: return QString("BTC");
-    case mBTC: return QString("mBTC");
-    case uBTC: return QString::fromUtf8("μBTC");
+    case BTC: return QString(CURRENCY_UNIT.c_str());
```

Comment:
Let's leave this function as it is. So anyone changing the currency unit will have to change `bitcoinunits` as well, that's OK IMO, that's what it was introduced for.
(This is concatenation is weird, and will mismatch the constant names)